### PR TITLE
Add support for AIDL (interfaces + parcelables)

### PR DESCRIPTION
**Pull request by bwalter**
- Add JsValue.fromAidlInterface()
- Add support for byte

### CODE REVIEW

File: jsbridge/src/main/jni/JavaTypeProvider.cpp [hunk 1]
```diff
@@ -65,10 +68,13 @@ const JavaType *JavaTypeProvider::newType(const JniRef<jsBridgeParameter> &param
     case JavaTypeId::Unknown:
       return nullptr;
     case JavaTypeId::Void:
+      return new Void(m_jsBridgeContext, id, false /*boxed*/);  // TODO: check it
```
Comment (oleksandr-yefremov) [inline jsbridge/src/main/jni/JavaTypeProvider.cpp:71]: Should this Todo be fixed before merging? Same for [another one](https://github.com/p7s1digital/oasis-jsbridge-android/pull/25/files#diff-43450cc29dc8b1056120b7b4eabd19bcf9fe10e598c57242e4394cf87bbe685eR138) down the code.

**Comment by oleksandr-yefremov** [comment]
Great addition 👍 @bwalter, just curious, do you need AIDL somewhere in your work or just had some free time? :)

File: jsbridge/src/main/jni/JavaTypeProvider.cpp (hunk 1 above)
Comment (bwalter) [reply jsbridge/src/main/jni/JavaTypeProvider.cpp]: Thanks for pointing it out! The change has already been checked and removed the comment. The other one way still requiring a change which has now been addressed :)

**Comment by bwalter** [comment]
> Great addition 👍 @bwalter, just curious, do you need AIDL somewhere in your work or just had some free time? :)

It is mainly a free time change to prove that the bridge is suitable for AIDL. I can see really great use cases (especially for work), though. It would be purely awesome to introduce scripting capabilities in a landscape of Android apps using AIDL to communicate with each other (including for testing).

**Pull request merged**